src/olmo_core/internal/common.py

```diff
@@ -109,7 +108,8 @@ def build_launch_config(
     workspace: str = "ai2/OLMo-core",
     budget: str = "ai2/oe-base",
     nccl_debug: bool = False,
-    beaker_image: str = OLMoCoreBeakerImage.stable,
+    cuda_launch_blocking: bool = False,
+    beaker_image: str = "olmo-core-tch280cu128-2025-09-24",
```

Comment:
We'll need to revert the beaker image to `stable` here. If the SFT script needs this specific image we can just pass it in directly from there.